migrations/2_dev_dependencies.js

```diff
@@ -28,7 +27,9 @@ module.exports = async function (deployer, network, accounts) {
 }
 
 async function deploySafe(deployer) {
```

Comment:
If we plan to move other items into here like Multisend, perhaps this function could use a rename `deploySafeItems`. Or is Multisend something that would continue to live outside?

Reply:
Done in #537